pkgs/development/python-modules/blip/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, fetchFromGitHub
+, timm
+, transformers
+, fairscale
+}:
+
+stdenv.mkDerivation {
+  pname = "blip";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "salesforce";
+    repo = "BLIP";
+    rev = "48211a1594f1321b00f14c9f7a5b4813144b2fb9";
+    sha256 = "sha256-0IO+3M/Gy4VrNBFYYgZB2CzWhT3PTGBXNKPad61px5k=";
+  };
+
+
+  buildPhase = ''
+    mkdir -p $out/lib/python3.10/site-packages
+    cp -R ./ $out/lib/python3.10/site-packages/blip/
```

Comment:
`lib/python3.10/site-packages` -> `${python.sitePackages}`, but can I suggest that instead of custom `buildPhase`s we just add our own [`pyproject.toml`](https://github.com/SomeoneSerge/pkgs/blob/df599c26db7404aafb95e9bf2182c87360463f0d/python-packages/by-name/sa/salesforce-blip/pyproject.toml) file in postPatch: https://github.com/SomeoneSerge/pkgs/blob/df599c26db7404aafb95e9bf2182c87360463f0d/python-packages/by-name/sa/salesforce-blip/package.nix#L34.

This way we can just set [`pyproject = true`](https://github.com/SomeoneSerge/pkgs/blob/df599c26db7404aafb95e9bf2182c87360463f0d/python-packages/by-name/sa/salesforce-blip/package.nix#L21) and avoid having any custom phases, and we'd also generate the `.dist-info` for our package.

We could also open the respective PRs to at least try to upstream the `pyproject.toml`s